.github/workflows/update-icons.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       - name: Check for changes
         id: check-icons
         run: |
-          echo modified=$(`git status --porcelain | grep -q *.svg` && echo true || echo false) >> $GITHUB_OUTPUT
+          echo modified=$(`git status --porcelain | grep -q .svg$` && echo true || echo false) >> $GITHUB_OUTPUT
```

Comment:
@fallaciousreasoning you might wanna replace the `.` with `\.` since the existing one matches any character, not necessarily a dot I believe